5e-SRD-Monsters.json

```diff
@@ -280,69 +350,109 @@
-        "attack_bonus": 0
+        "desc": "The dracolich can use its Frightful Presence. It then makes three attacks: one with its bite and two with its claws."
       },
       {
         "name": "Bite",
         "desc": "Melee Weapon Attack: +12 to hit, reach 10 ft., one target. Hit: 18 (2d10 + 7) piercing damage plus 5 (1d10) lightning damage.",
         "attack_bonus": 12,
-        "damage_dice": "2d10 + 1d10",
-        "damage_bonus": 7
+        "damage": [
+          {
+            "damage_type": "piercing",
+            "damage_dice": "2d10",
+            "damage_bonus": 7
+          },
+          {
+            "type": "lightning",
+            "dice": "1d10",
+            "bonus": 0
+          }
+        ]
       },
       {
         "name": "Claw",
         "desc": "Melee Weapon Attack: +12 to hit, reach 5 ft., one target. Hit: 14 (2d6 + 7) slashing damage.",
         "attack_bonus": 12,
-        "damage_dice": "2d6",
-        "damage_bonus": 7
+        "damage": [
+          {
+            "damage_type": "slashing",
+            "damage_dice": "2d6",
+            "damage_bonus": 7
+          }
+        ]
       },
       {
         "name": "Tail",
         "desc": "Melee Weapon Attack: +12 to hit, reach 15 ft., one target. Hit: 16 (2d8 + 7) bludgeoning damage.",
         "attack_bonus": 12,
-        "damage_dice": "2d8",
-        "damage_bonus": 7
+        "damage": [
+          {
+            "damage_type": "bludgeoning",
+            "damage_dice": "2d8",
+            "damage_bonus": 7
+          }
+        ]
       },
       {
         "name": "Frightful Presence",
         "desc": "Each creature of the dracolich's choice that is within 120 feet of the dracolich and aware of it must succeed on a DC 18 Wisdom saving throw or become frightened for 1 minute. A creature can repeat the saving throw at the end of each of its turns, ending the effect on itself on a success. If a creature's saving throw is successful or the effect ends for it, the creature is immune to the dracolich's Frightful Presence for the next 24 hours.",
-        "attack_bonus": 0
+        "dc": {
+          "dc_type": "WIS",
+          "dc_value": 18,
+          "success_type": "none"
+        }
       },
       {
         "name": "Lightning Breath (Recharge 5-6)",
         "desc": "The dracolich exhales lightning in a 90-foot line that is 5 feet wide. Each creature in that line must make a DC 20 Dexterity saving throw, taking 66 (12d10) lightning damage on a failed save, or half as much damage on a successful one.",
-        "attack_bonus": 0,
-        "damage_dice": "12d10"
+        "dc": {
+          "dc_type": "DEX",
+          "dc_value": 20,
+          "success_type": "half"
+        },
+        "damage": [
+          {
+            "damage_type": "lightning",
+            "damage_dice": "12d10",
+            "damage_bonus": 7
+          }
```

Comment:
just thinking about the DC on this, as its applied before any damage where as the some DCs for attacks are made, such as line 59. I don't know if it needs to be differentiated but just a thought

Reply:
I've actually been thinking about that as well. I've only been doing things where DC is applied to damage as is. I'm intentionally avoiding DC for things that happen after damage, like additional status affects. Some of those additional affects are not regular status affects either. Probably how I'll define them is part of the `damage` array but I'm taking any suggestions.

Reply:
@Alex-frazer It would probably be a bit of extra work, but do you think I should try and work in the other kinds of DC? It would probably look something like this:
```json
          {
            "name": "Cold Breath",
            "dc": {
              "dc_type": "CON",
              "dc_value": 20,
              "success_type": "half"
            },
            "damage": [
              {
                "damage_type": "cold",
                "damage_dice": "13d8",
                "damage_bonus": 0
              },
              {
                "dc_type": "CON",
                "dc_value": 20,
                "success_type": "half"
              },
            ]
          },
```
unless I can find a better shape.

Reply:
having it twice seems quite confusing to me, I can do this on a later branch after you commit unless there is some thing that is quite clear

Reply:
unless there are 2 saves for cold breath, can't remember off the top of my head

Reply:
There aren't two saves for Cold Breath. But I believe I've run across some abilities so far where there are multiple DC checks in one ability. If I run into one, I'll post it here.

But otherwise, I think I'll punt on this for you to try on a later branch unless I have a great idea.

Reply:
There is stuff like this:

> "Ranged Weapon Attack: +4 to hit, range 30/120 ft., one target. Hit: 5 (1d6 + 2) piercing damage, and the target must succeed on a DC 13 Constitution saving throw or be poisoned for 1 hour. If the saving throw fails by 5 or more, the target is also unconscious while poisoned in this way. The target wakes up if it takes damage or if another creature takes an action to shake it awake."